mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -728,7 +728,7 @@ def dict_repr(self):
 class DetectorDistanceHOMediator(HOMediatorBase):
     def __init__(self, ho, name=''):
         super(DetectorDistanceHOMediator, self).__init__(ho, name)
-        ho.connect("positionChanged", self.value_change)
+        #ho.connect("positionChanged", self.value_change)
```

Comment:
I have not seen anything on our side, probably because for us dtox is a SardanaMotor. I totally agree on having separate PRs, better that way for everyone ;)